content/tracing/getting_further/apm_events.md

```diff
@@ -0,0 +1,64 @@
+---
+title: APM Events
+kind: Documentation
+further_reading:
+- link: "tracing/search"
+  tag: "Documentation"
+  text: Global search of all your traces with tags
+- link: "tracing/analytics"
+  tag: "Documentation"
+  text: Analytics on your APM data at infinite cardinality
+---
+
+## What are APM Events?
+
+An APM event is generated every time a trace is generated, it correspond to all the metadata of the trace plus the [top span][3] of the trace. 
+
+Why APM events are not just trace? because traces [get sampled][4] and APM event don't, and we only keep the top span information for the APM event, not the full trace.
```

Comment:
`Why aren't APM events just traces? Because traces [get sampled][4] and APM events don't, and we only keep the top span information for the APM event, not the full trace.`